### PR TITLE
4.7 Docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -309,6 +309,18 @@ Enable Team Directory
...
+Allows for leave/join messages to be created in the default channel.
+
+**True**: Leave/join messages can be created in the default channel.
+
+**False**: Leave/join messages will not be created in the DB for the default channel.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableDefaultChannelLeaveJoinMessages": true`` with options ``true`` and ``false`` for above settings respectively.          |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
```
Comment (amyblais) [inline source/administration/config-settings.rst]: For some reason the formatting here doesn't work (when I preview).

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2469,6 +2481,15 @@ Vary rate limiting by HTTP header field specified (e.g. when configuring Ngnix s
 | This feature's ``config.json`` setting is ``"VaryByHeader": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Vary rate limit by user
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**True**: // XXXX Need help here
+
+**False**: 
+
```
Comment (amyblais) [inline source/administration/config-settings.rst]: I need help with the description part.

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -309,6 +309,18 @@ Enable Team Directory
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Allows for leave/join messages to be created in the default channel.
+
+**True**: Leave/join messages can be created in the default channel.
+
+**False**: Leave/join messages will not be created in the DB for the default channel.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableDefaultChannelLeaveJoinMessages": true`` with options ``true`` and ``false`` for above settings respectively.          |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (amyblais) [inline source/administration/config-settings.rst]: This formatting again doesn't work in the preview.

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -3413,6 +3434,17 @@ Enable Preview Features (Experimental)
...
+Group Unreads Channels (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: Show unreads channels in the User Interface.
+
+**False**:  Hides unreads channels in the User Interface and disables this feature for the server.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature’s ``config.json`` setting is ``"ExperimentalGroupUnreadChannels": false`` with options ``true`` and ``false`` for above settings respectively.                     |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (amyblais) [inline source/administration/config-settings.rst]: The formatting here again doesn't work in the preview.

**Comment by amyblais** [comment]
Will need some help ensuring the info in the Image Proxy doc is accurate.

File: source/administration/image-proxy.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
+Image Proxy
+================================
+
+Using image proxies means that users make only secure requests. By allowing image requests to go straight to third-party
+servers, tracking pixels is allowed. Users can put invisible images in posts that logs time and location data
+for every user that views the post. An image proxy protects user privacy by eliminating this direct interaction with 
+third-party servers.
+
+Many proxy servers also provide resizing-as-a-service. For example, if you use `willnorris/imageproxy`, you can configure
```
Comment (lindy65) [inline source/administration/image-proxy.rst]: I had to Google who/what "willnorris" is...  Would it clarify to end-users if we included the full GitHub address e.g. "https://github.com/willnorris/imageproxy" so they realise it's image proxy software written by the person "Will Norris"?

**Comment by jasonblais** [comment]
@amyblais We may need to update the image proxy doc slightly due to https://github.com/mattermost/mattermost-webapp/pull/777

We support `atmos/camo` proxy service instead with sample settings on pre-release (although I wouldn't hide/blur the URL setting)

File: source/administration/config-settings.rst [hunk 6]
```diff
@@ -309,6 +309,18 @@ Enable Team Directory
 | This feature's ``config.json`` setting is ``"EnableTeamListing": false`` with options ``true`` and ``false`` for above settings respectively.                        |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Default Channel Leave/Join Messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Allows for leave/join messages to be created in the default channel.
```
Comment (esethna) [inline source/administration/config-settings.rst]: Can we clarify a bit more about what this does? Are these team join leave messages in town square that are being enabled/disabled?

File: source/administration/config-settings.rst (hunk 1 above)
Comment (esethna) [reply source/administration/config-settings.rst]: Fixed. The table had extra spaces in it

File: source/administration/config-settings.rst [hunk 7]
```diff
@@ -2133,6 +2145,15 @@ When false, disables file downloads on mobile apps. Users can still download fil
 | This feature's ``config.json`` setting is ``"EnableMobileDownload": true`` with options ``true`` and ``false``.     |
 +---------------------------------------------------------------------------------------------------------------------+
 
+Image Proxy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Includes three configuration keys: `ImageProxyType`, `ImageProxyURL` and `ImageProxyOptions`. When these keys are configured, posts served to the client will have their markdown modified enabling all images to be loaded through a proxy.
```
Comment (esethna) [inline source/administration/config-settings.rst]: Propose link to docs explaining more?

File: source/administration/config-settings.rst (hunk 2 above)
Comment (esethna) [reply source/administration/config-settings.rst]: Please reach out to JB

File: source/administration/image-proxy.rst (hunk 5 above)
Comment (esethna) [reply source/administration/image-proxy.rst]: Didn't we remove support for willnorris in 4.7? Maybe remove it from the docs as an example if so? (JB)

**Comment by jasonblais** [comment]
Merging after a few small tweaks.

@amyblais In-line code in RST requires two backticks (``) instead of one (`).

@amyblais Can you ask Chris Brown to review the image proxy documentation tomorrow? For technical accuracy.

**Pull request merged**